src/server/db_slice.cc

```diff
@@ -266,6 +273,11 @@ auto DbSlice::GetStats() const -> Stats {
   return s;
 }
 
+SlotStats DbSlice::GetSlotStats(SlotId sid) const {
+  CHECK(db_arr_[0]);
```

Comment:
Do we only ever care about db 0?

Reply:
Redis supports only db 0 in cluster mode. 
We should check when loading redis cluster what is the error that is returned when using select 1 and also add this in our flow.

Reply:
Gotcha, that's a good explanation, thanks